### PR TITLE
Update ns-timeline.md

### PR DESCRIPTION
**Pull request by aysebkaya**
This is done as a first draft, there are things that need adding and changing.

### CODE REVIEW

File: docs/components/ns-timeline.md [hunk 1]
```diff
@@ -66,43 +75,75 @@ You can also add a summary to manage expectations
...
+
+#### Component placement
+
+<ComponentPlacement component="ns-timeline" parentComponents="ns-panel"></ComponentPlacement>
+
+<ComponentPlacement component="ns-timeline-event" parentComponents="ns-timeline"></ComponentPlacement>
+
+### Step
+
+The step variant is a simpler version of the timeline component that excludes summary and status.
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:97]:
```suggestion
The step variant is a simpler version of the timeline component that excludes summary and status. It is better suited to set the expectation of a process with a number of steps if an action is taken by a user. 

```

File: docs/components/ns-timeline.md [hunk 2]
```diff
@@ -66,43 +75,75 @@ You can also add a summary to manage expectations
...
+
+In the step variant, additional information can always be added to an event if more explanation is needed. 
+
+#### Usage
+
+Coming soon...
+
+#### Component placement
+
+Coming soon...
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:123]: If that is the same as the main timeline, I would suggest having that only once for the overall component :)

File: docs/components/ns-timeline.md [hunk 3]
```diff
@@ -66,43 +75,75 @@ You can also add a summary to manage expectations
...
+
 ## Best practice for Timeline
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:125]: A couple changes to make the best practice section its own with two sub sections

```suggestion
## Best practice
```

File: docs/components/ns-timeline.md [hunk 4]
```diff
@@ -66,43 +75,75 @@ You can also add a summary to manage expectations
...
 | 💚 Do's | 💔 Don'ts |
 | :--- | :--- |
 | Use a maximum of 10 steps | Add too much detailed content |
 | Consider if the title duplicates page title | Use illustrations or icons in additional content |
 | Provide helpful messaging | Use overly long event names |
-| Use summary to set expectations | Use timestamp if it causes confusion with a date elsewhere on the page|
+| Use summary to set expectations for the `standard` variant | Use timestamp if it causes confusion with a date elsewhere on the page for the `standard` variant |
 
-## Best practice for Individual Timeline Event
+## Best practice for Individual Timeline Event
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:134]:
```suggestion
### Step variant best practice
```

File: docs/components/ns-timeline.md [hunk 5]
```diff
@@ -66,43 +75,75 @@ You can also add a summary to manage expectations
...
+
+#### Usage
+
+Coming soon...
+
+#### Component placement
+
+Coming soon...
+
 ## Best practice for Timeline
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:126]: ### Standard variant best practice

File: docs/components/ns-timeline.md [hunk 6]
```diff
@@ -66,43 +75,75 @@ You can also add a summary to manage expectations
...
-## Component placement
-
-<ComponentPlacement component="ns-timeline" parentComponents="ns-panel"></ComponentPlacement>
-
-<ComponentPlacement component="ns-timeline-event" parentComponents="ns-timeline"></ComponentPlacement>
+| Use text or `ns-card` within `additional content` to provide additional information | Add status to an event for the `step` variant |
+| Use either default, icon or number decoration for all events in the timeline sequence for the `standard` variant |  Mix two different decoration types in events |
+| Use either number or fallback decoration for all events in the timeline sequence for the `Step` variant | Use any icon type with names ending `solid` or `outline` |
 
 ## Specification for Timeline
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:142]: Setting he same section structure as for the best practice :)

```suggestion
## Specification

### Main component specification
```

Comment (PtitBen) [reply docs/components/ns-timeline.md:142]: I couldn't add a suggestion, but I would make the `## Specification for Individual Timeline Event` a h3 so it's a sub-section too.

**Review by PtitBen — COMMENTED** [review]
hope those make sense :)

File: docs/components/ns-timeline.md (hunk 2 above)
Comment (aysebkaya) [reply docs/components/ns-timeline.md:123]: When the changes are live Rob and I will be able to embed the component into the documentation.

File: docs/components/ns-timeline.md [hunk 7]
```diff
@@ -9,16 +9,33 @@ import { PageFooter } from '../../includes/page-footer.js'
...
+> The Timeline component represents a sequence of events that happen within a process.
 
 It gives an overview of the whole process and progress of a customer journey. Its main purpose is to create situational awareness for the user and manage expectations.
 
+
+## Content guidance
+
+### Variants and Purpose
+
+There are two variants of the timeline component.
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:21]:
```suggestion

- standard version: to show steps of a process that has started, allowing to show the status of each step
- step version: to set the expectation of a process that hasn't started or doesn't have a specific status
```

File: docs/components/ns-timeline.md [hunk 8]
```diff
@@ -66,43 +84,57 @@ You can also add a summary to manage expectations
...
+
+#### Heading
+
+There is provision for a heading to describe what process the timeline is showing. This can be left blank if it has already been made clear in an ns-landmark above it to avoid duplication.
+
+#### Showing additional information
+
+In the step variant, additional information can always be added to an event if more explanation is needed. 
+
+## Best practice
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:116]:
```suggestion
## Best practice for the timeline component
```

File: docs/components/ns-timeline.md [hunk 9]
```diff
@@ -66,43 +84,57 @@ You can also add a summary to manage expectations
...
 | 💚 Do's | 💔 Don'ts |
 | :--- | :--- |
 | Use a maximum of 10 steps | Add too much detailed content |
 | Consider if the title duplicates page title | Use illustrations or icons in additional content |
 | Provide helpful messaging | Use overly long event names |
-| Use summary to set expectations | Use timestamp if it causes confusion with a date elsewhere on the page|
+| Use summary to set expectations for the `standard` variant | Use timestamp if it causes confusion with a date elsewhere on the page for the `standard` variant |
 
-## Best practice for Individual Timeline Event
+## Best practice for Individual Timeline Event
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:125]:
```suggestion
## Best practice for the individual timeline events
```

File: docs/components/ns-timeline.md [hunk 10]
```diff
@@ -66,43 +84,57 @@ You can also add a summary to manage expectations
...
-## Component placement
-
-<ComponentPlacement component="ns-timeline" parentComponents="ns-panel"></ComponentPlacement>
-
-<ComponentPlacement component="ns-timeline-event" parentComponents="ns-timeline"></ComponentPlacement>
+| Use text or `ns-card` within `additional content` to provide additional information | Add status to an event for the `step` variant |
+| Use either default, icon or number decoration for all events in the timeline sequence for the `standard` variant |  Mix two different decoration types in events |
+| Use either number or fallback decoration for all events in the timeline sequence for the `Step` variant | Use any icon type with names ending `solid` or `outline` |
 
 ## Specification for Timeline
```
Comment (PtitBen) [inline docs/components/ns-timeline.md:133]:
```suggestion
## Specification for the Timeline component
```

**Review by PtitBen — COMMENTED** [review]
I made some final suggestions, if you could commit them? 

And one thing I couldn't suggest, can you change 

`# Specification for Individual Timeline Event`

for 

`# Specification for the individual timeline events`

After that, I'll be happy to approve the change!

**Comment by PtitBen** [comment]
As a note, for headers: we do not use title casing (capitalising each word), use sentence casing instead :)

**Comment by PtitBen** [comment]
Hey @aysebkaya , can you please use sentence casing for the headers? 
(check the suggestions I made for reference. We do not capitalise the words component or timeline, or event(s) there. 

Thanks

**Review by PtitBen — APPROVED** [review]

**Pull request merged**